templates/store/publisher-details.html

```diff
@@ -0,0 +1,212 @@
+    <div class="row">
+      <div class="u-equal-height">
+        {% set snap = featured_snaps[0] %}
+        <div class="col-6 p-featured-snap" style="background-color:{{ snap.background }}">
+          <a class="p-media-object--snap u-no-margin--bottom" href="/{{ snap.package_name }}">
+            {% if snap.icon_url %}
+              <img src="{{ snap.icon_url }}" class="p-media-object--snap__img" alt="">
+            {% else %}
+              <img src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" class="p-media-object--snap__img" alt="">
+            {% endif %}
+
+            <div class="p-media-object--snap__content">
+              <h4 class="p-media-object--snap__title">
+                {{ snap.title }}
+              </h4>
+            </div>
+            <div class="p-media-object--snap__description">
+              <hr />
+              <h5>{{ snap.summary }}</h5>
+              <p>{{ snap.description }}</p>
+            </div>
+          </a>
+        </div>
+        {% if featured_snaps[1] %}
+        {% set snap = featured_snaps[1] %}
+        <div class="col-6 p-featured-snap" style="background-color:{{ snap.background }}">
+          <a class="p-media-object--snap u-no-margin--bottom" href="/{{ snap.package_name }}">
+            {% if snap.icon_url %}
+              <img src="{{ snap.icon_url }}" class="p-media-object--snap__img" alt="">
+            {% else %}
+              <img src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" class="p-media-object--snap__img" alt="">
+            {% endif %}
+
+            <div class="p-media-object--snap__content">
+              <h4 class="p-media-object--snap__title">
+                {{ snap.title }}
+              </h4>
+            </div>
+            <div class="p-media-object--snap__description">
+              <hr />
+              <h5>{{ snap.summary }}</h5>
+              <p>{{ snap.description }}</p>
+            </div>
+          </a>
+        </div>
+        {% endif %}
+      </div>
+    </div>
+  </div>
+  {% endif %}
+
+  <div class="p-strip is-shallow">
+    <div class="row">
+      <div class="col-12 p-fluid-grid__publisher-snaps">
+      {% for snap in snaps %}
+        <div class="p-fluid-grid__item">
+          <a class="p-media-object--snap{% if snap_no_margin %}u-no-margin--bottom{% endif %}" href="/{{ snap.package_name }}">
+            {% if snap.icon_url %}
+            <img src="{{ snap.icon_url }}" class="p-media-object--snap__img" alt="">
+            {% else %}
+            <img src="https://assets.ubuntu.com/v1/be6eb412-snapcraft-missing-icon.svg" class="p-media-object--snap__img" alt="">
+            {% endif %}
+
+            <div class="p-media-object--snap__content">
+              <h4 class="p-media-object--snap__title">
+                {{ snap.title }}
+              </h4>
+              <div class="p-media-object--snap__desc">
+                <p>{{ snap.summary }}</p>
```

Comment:
Can we update `templates/store/_media-object-snap-partial.html` to handle this small variation?

```
// templates/store/_media-object-snap-partial.html
{% if snap_show_summary %}
  <p>{{ snap.summary }}</p>
{% else %}
  ...the current <p> in that template
{% endif %}
```

```
// publisher-details.html
{% for snap in snaps %}
  {% set snap_show_summary=true %}
  {% include("store/_media-object-snap-partial.html") %}
{% endfor %}
```

Reply:
👍